client/src/components/ui/RowTemplate.svelte

```diff
@@ -6,38 +6,55 @@
     import OverflowMenuVertical from '../icons/OverflowMenuVertical.svelte';
 
     export let iconSize: IconSize = IconSize.Large;
-    export let title: string;
+    export let showOverflowIcon = true;
 
     const dispatch = createEventDispatcher();
 </script>
 
-<article>
+<article on:keydown on:click|stopPropagation={() => dispatch(Events.RowContainerClick)}>
     <div class="icon"><slot name="icon" /></div>
-    <p>{title}</p>
-    <div class="icon" on:keydown on:click|stopPropagation={() => dispatch(Events.OverflowClick)}>
-        <OverflowMenuVertical size={iconSize} alt="Show overflow menu" />
-    </div>
+    <div class="header"><slot/></div>
+    <div class="subtext"><slot name="secondary"/></div>
+    {#if showOverflowIcon}
+        <div class="overflow" on:keydown on:click|stopPropagation={() => dispatch(Events.OverflowClick)}>
+            <OverflowMenuVertical size={iconSize} alt="Show overflow menu" />
+        </div>
+    {/if}
 </article>
 
 <style>
     @import url('../../pages/vars.css');
 
     article {
-        display: flex;
+        display: grid;
```

Comment:
Using CSS Grid seems a little bit of an overkill here, where Flexbox is sufficient. Although the layout does work, we should consider simplifying. Here is my idea:

```html
<!-- Note the annotations for what were originally `grid-area` assignments. -->

<section>
    <div>a</div>
    <div id="middle">
        <!-- Two block-level siblings so they take up the whole width. -->
        <div id="b"></div>
        <div id="d"></div>
    </div>
    <div>c</div>
</section>
```

```css
section {
    display: flex;
}

section > div {
    /** Make sure every immediate child shrinks! */
    flex: 0 1 auto; /** already default */
}

#middle {
    /** Exception: make the middle expand as large as possible. */
    flex: 1 0 auto;
}
```